packages/nextra-theme-docs/src/utils/get-git-issue-url.ts

```diff
@@ -17,7 +17,7 @@ export const getGitIssueUrl = ({
       repo.name
     }/-/issues/new?issue[title]=${encodeURIComponent(title)}`
   } else if (repo.resource.includes('github')) {
-    return `${repo.resource}/${repo.owner}/${
+    return `https://github.com/${repo.owner}/${
```

Comment:
The problem now is that the domain is static for issues. I think using `repo.protocol` + `repo.source` will solve this for GitHub and GitLab. What do you think?

Reply:
I think currently is fine

Reply:
Hm, what I wanted to achieve is that self-hosted GitHub enterprise instances do work too here. But with the static GitHub url, this does not work.

For example, we have instances under github.acme.co, so the solution with the host works, as same as for public GitHub.com.

That's why I am proposing a change here with the way the url is build

Reply:
Done, see latest commit